src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -792,6 +806,26 @@ cli_drop_node_getopts(int argc, char **argv)
 				break;
 			}
 
+			case 'n':
```

Comment:
user should not be able to specidy --pgdata when --nodename or --nodeport is specified, also we should probably require --nodeport if --nodename is specified

Reply:
We still need the PGDATA of the local monitor node, when dropping a distant node. So PGDATA is either the one of the node that is being removed or destroyed, or the PGDATA of the monitor where we want to call `select pgautofailover.remove_node(nodename, pgport)`.